src/lazy-image.component.ts

```diff
@@ -156,6 +160,11 @@ export class LazyImageComponent implements AfterViewInit, OnInit {
                 this.imageHeight = image.height;
                 this.updateStretchState();
             });
+
+            image.addEventListener('error', () => {
+                this.loading = false;
+                this.errorOccurred = true;
```

Comment:
would be nice to emit some output here... in that case component using it will be able to do some special action if needed...

Reply:
Yep, improvement idea for the next v.